deeds-dapp-service/src/main/java/io/meeds/deeds/service/MeedTokenMetricService.java

```diff
@@ -172,6 +188,14 @@ public Map<String, BigDecimal> getLockedBalances() {
     return lockedBalances;
   }
 
+  public BigDecimal getTotalLockedBalance(Map<String, BigDecimal> lockedBalances) {
+    return lockedBalances.values().stream().reduce(BigDecimal::add).orElse(BigDecimal.valueOf(0));
+  }
+
+  public BigDecimal getTotalReserveBalance(Map<String, BigDecimal> reserveBalances) {
```

Comment:
```suggestion
  private BigDecimal getTotalReserveBalance(Map<String, BigDecimal> reserveBalances) {
```
utility internal method, not for Service API